search/src/main/java/edu/artic/search/SearchBaseViewModel.kt

```diff
@@ -94,13 +93,20 @@ open class SearchBaseViewModel @Inject constructor(
                 navigateTo.onNext(Navigate.Forward(NavigationEndpoint.Web))
             }
             is SearchCircularCellViewModel -> {
-                viewModel.artWork?.let { articObject ->
-                    navigateTo.onNext(
-                            Navigate.Forward(
-                                    NavigationEndpoint.ArtworkOnMap(articObject)
-                            )
-                    )
+                viewModel.artWork?.let { arcticObject ->
+                    /* Convert the ArcticObject to ArcticSearchArtworkObject*/
+                    val arcticSearchArtworkObject = ArticSearchArtworkObject(
+                            artworkId = arcticObject.id.toString(),
+                            audioObject = arcticObject,
+                            title = arcticObject.title,
+                            thumbnailUrl = arcticObject.thumbUrl,
+                            imageUrl = arcticObject.largeImageUrl,
+                            artistDisplay = arcticObject.artistCulturePlaceDelim,
+                            location = arcticObject.location,
+                            floor = arcticObject.floor,
+                            gallery = null)
```

Comment:
Put close parenthesis `)` on its own line.

Reply:
add a comment too saying that we don't pass 'gallery' because we don't need it at this time, or something like that.